cloud/gcp/permissions.tf

```diff
@@ -5,12 +5,15 @@ resource "google_project_iam_custom_role" "sfx_role" {
   title       = "SignalFx${var.suffix == "" ? "" : " - ${title(var.suffix)}"}"
   description = "SignalFx viewer role for monitoring${var.suffix == "" ? "" : " for ${lower(var.suffix)}"}"
   permissions = [
+    "compute.instances.list",
```

Comment:
Please keep the order from https://docs.splunk.com/Observability/gdi/get-data-in/connect/gcp/gcp.html#gcp-one